features/OCM_disable_rule_on_ACM.feature

```diff
@@ -0,0 +1,117 @@
+Feature: Disabling rule in OCM user interface should NOT be visible in ACM UI
+
+
+  Scenario: Check if OCM user interface is accessible for two test accounts
```

Comment:
I would rename this with "check if log in works for two test account" or something similiar.

because 

Check if OCM user interface is accessible

and then 

Given OCM user interface is accessible

means semantically that there's nothing to check?

Reply:
yup, I'll change it!